reyna/src/com/b2msolutions/reyna/Repository.java

```diff
@@ -92,13 +97,18 @@ public void shrinkDb(long limit) {
             limit -= SIZE_DIFFERENCE_TO_START_CLEANING;
             long dbSize = this.getDbSize(db);
 
+            Logger.v(TAG, "shrinkDb, dbSize: " + dbSize);
+            Logger.v(TAG, "shrinkDb, limit: " + limit);
             if (dbSize <= limit) {
+                Logger.v(TAG, "shrinkDb, dbSize <= limit, no SHRINK");
                 return;
             }
 
+            Logger.v(TAG, "shrinkDb, dbSize <= limit, SHRINK needed");
```

Comment:
Should be "shrinkDb, dbSize > limit, SHRINK needed". Also as we can shrink multiple time within the loop, should we put this log message inside the loop as well?